collections/_article/funding-breakdown-and-hiring-process.md

```diff
@@ -0,0 +1,102 @@
+
+Thanks to this policy, Godot development can happen at a fraction of the cost of larger commercial offerings and get more done dollar-for-dollar (and making the results fully free, Open Source, and available for everyone to use in any way they want).
+
+This approach to hiring has scaled well with the growth of the project and we hope that it continues into the future. As always, we strive to do as much as we can with what we have and we hope that you continue to trust us and believe in the future of the project.
+
+## Future of funding
+
+### New donation platform
+
+We are launching a [Godot Development Fund page](https://fund.godotengine.org/), taking inspiration from our friends at Blender and the amazing Blender Development Fund. The Godot Development Fund will operate much the same way allowing users to make recurring or one-time donations from a centralized location.
+
+We decided on offering this new donation platform as it became increasingly clear that using a third party tool (like Patreon) was costing too much relative to the service it offers (with taxes, payment processing fees and other costs, Godot is receiving substantially less than what is donated through the platform). At the same time, there are few similar services that cater to not for profits operating on the scale of Godot. Ultimately, we have decided to follow in the footsteps of our friends at Blender and Krita.
+
+The new donation platform is mostly ready and you can already use it to set up a recurring donation via PayPal if you would like! In the coming weeks we expect to finalize the donation platform and we will have a big announcement post once it is 100% finished.
+
+We will keep the Patreon operational for the foreseeable future as we heavily rely on the current donations we receive through it. But for future donations please consider using the Development Fund as it results in more of your donation going towards development.
+
+### Diversification of funding sources
+
+With the new Godot Foundation, it is expected that the Godot Project should be able to further diversify its funding, going into areas such as:
+
+- Paid asset store (taking a % of sales).
+- Merchandise.
+
+But these are still in the early planning phases as our priority until now has been to get the donation platform operational. To be clear, we do not anticipate that we will be able to sustain the project through the asset store and merchandise alone as we will intentionally operate them with very thin margins. We are a not-for-profit project first and foremost and we will not compromise that.
```

Comment:
I'm not sure if I'm convinced about the last `;`. I'm not sure if it is because I'm not that good at using them, but I believe the `and` is better in this case. If we split the sentence into two the second one wouldn't make sense without the first one.

Reply:
Are you okay with Clay's edit @Calinou ?

Reply:
> Are you okay with Clay's edit @Calinou ?

Sounds good 🙂